x-team/xp-ui
# Fix Milestones responsiveness on SignupScreen

**Release Type:** *Bug-fix*

Fixes https://x-team-internal.atlassian.net/browse/XP-3802

## Description

Fix Milestones responsiveness on SignupScreen

## Checklist


## Steps to Test or Reproduce

1. Go to http://localhost:9001/?path=/story/screens-and-layouts-signupscreen--basic-usage
1. Resize the `SignupScreen` viewport to a small size and observe how it looks
1. Also resize the viewport to a size bigger than `500px` and observe how it looks

![2020-07-02 12 01 12](https://user-images.githubusercontent.com/131859/86376488-70487a80-bc5d-11ea-9bcb-8c85c365df64.gif)


## Impacted Areas in Application

`src/components/ui/SignupScreen.js`
